ci-operator/config/openshift-pipelines/release-tests/OWNERS

```diff
@@ -8,8 +8,10 @@ approvers:
 - openshift-cherrypick-robot
 - ppitonak
 - veeresharadhya
+- srivickynesh
```

Comment:
Please see the very first line of this file :) You need to edit https://github.com/openshift-pipelines/release-tests/blob/master/OWNERS and it will be synced at some point

Reply:
I will close this PR